.eslintrc.js

```diff
@@ -13,5 +13,9 @@ module.exports = {
     es6: true
   },
   rules: {
+    'arrow-parens': ['error', 'as-needed'],
+    // Destructuring is not supported in node 4.8.0
+    'ember-suave/prefer-destructuring': 'off',
+    'operator-linebreak': ['error', 'before', { 'overrides': { '=': 'ignore'} } ]
```

Comment:
I'm willing to talk this over if I'm missing a reason this might be desired, but currently I absolutely hate this rule